MANIFEST.in

```diff
@@ -1,4 +1,8 @@
 recursive-include panoramix/templates *
 recursive-include panoramix/static *
+recursive-exclude panoramix/static/assets/node_modules *
```

Comment:
gotcha, and they're already included in the bundled js files so are totally unnecessary